datafusion/substrait/tests/testdata/tpch/nation.csv

```diff
@@ -1,2 +1,2 @@
-n_nationkey,n_name,n_regionkey,n_comment
+N_NATIONKEY,N_NAME,N_REGIONKEY,N_COMMENT
 0,ALGERIA,0, haggle. carefully final deposits detect slyly agai
```

Comment:
@Blizzara this is based on what you did in this PR to fix the tests that fail due to case difference in fields name.